frameworks/react-legacy/src/awkWrapper.tsx

```diff
@@ -8,21 +8,22 @@ import WanderStrategy from "@arweave-wallet-kit/wander-strategy";
 import WebWalletStrategy from "@arweave-wallet-kit/webwallet-strategy";
 import OthentStrategy from "@arweave-wallet-kit/othent-strategy";
 import BrowserWalletStrategy from "@arweave-wallet-kit/browser-wallet-strategy";
-import BeaconWallet from "@vela-ventures/aosync-strategy";
+//import BeaconWallet from "@vela-ventures/aosync-strategy";
+import { Strategy } from "@arweave-wallet-kit/core/strategy";
 
 // Define the default strategies that will always be used
 const defaultStrategies = [
   new WanderStrategy(),
   new WebWalletStrategy(),
   new OthentStrategy(),
   new BrowserWalletStrategy(),
-  new BeaconWallet(),
+  // new BeaconWallet(),
```

Comment:
I recommend if beacon wallet is going to be a default strategy, it should be _in_ this monorepo. I was forced to comment it out here due to type conflicts, as well as strategy conflicts due to me adding some things to the strategy interface.